### PR TITLE
49986 : Badges display on session expired.

### PR DESCRIPTION
**Pull request by wbenabdessalem**
iOS_Session_expired_US01:  Badges display on session expired.

### CODE REVIEW

File: eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift [hunk 1]
```diff
@@ -118,7 +118,24 @@ extension ConnectToExoViewController:UITableViewDelegate,UITableViewDataSource{
         cell.setupDataWith(serveur:serveur)
         cell.deleteButton.tag = indexPath.row
         cell.deleteButton.addTarget(self, action: #selector(deleteButtonTapped(_ :)), for: .touchUpInside)
-        cell.badgeView.isHidden = true
+        // Handle Badge Notififcation appearance in case of session timeout
+        if let _quitTimestamp = defaults.value(forKey: "quitTimestamp") as? Double {
```
Comment (ahamdi) [inline eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift]: This is not a valid check for session timeout. The best will be to use this URL to get the user status https://url.of.exo/portal/rest/state/status/${userName}
could you use this to check the user connectivity ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift [hunk 2]
```diff
@@ -118,7 +118,24 @@ extension ConnectToExoViewController:UITableViewDelegate,UITableViewDataSource{
...
+            if (Date().timeIntervalSince1970 - _quitTimestamp) > eXoAppDelegate.sessionTimeout  {
```
Comment (ahamdi) [inline eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift]: eXoAppDelegate.sessionTimeout is not a right value for session timeout, we need to remove it completely

File: eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift (hunk 1 above)
Comment (wbenabdessalem) [reply eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift]: This link is not accessible through the web

File: eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift (hunk 2 above)
Comment (wbenabdessalem) [reply eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift]: Done ; with new check through the remote url

**Pull request closed without merging**